users/api/endpoints.go

```diff
@@ -112,12 +113,13 @@ func listUsersEndpoint(svc users.Service) endpoint.Endpoint {
 
 		res := usersPageRes{
 			pageRes: pageRes{
-				Total:  page.Total,
 				Offset: page.Offset,
 				Limit:  page.Limit,
+				Total:  page.Total,
 			},
 			Users: []viewUserRes{},
 		}
+
 		for _, user := range page.Users {
 			res.Users = append(res.Users, viewUserRes{User: user})
 		}
```

Comment:
Please revert back to these lines to `main` branch version
```suggestion
		res := usersPageRes{
			pageRes: pageRes{
				Total:  page.Total,
				Offset: page.Offset,
				Limit:  page.Limit,
			},
			Users: []viewUserRes{},
		}
		for _, user := range page.Users {
			res.Users = append(res.Users, viewUserRes{User: user})
		}
```